docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       ADMIN_PASSWORD: "password"
       ADMIN_EMAIL: "admin@example.com"
       DATABASE_URL: "postgres://doccano:doccano@postgres:5432/doccano?sslmode=disable"
+      ALLOW_SIGNUP: "True" 
```

Comment:
As a default value, I think "False" is better than "True" because it prevents unexpected signup.

Reply:
Fine for me